cypresse2e/cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+    "baseUrl": "http://localhost:3000/",
```

Comment:
should the baseUrl be the localhost or http://staging.fightpandemics.work/?

Reply:
Hi @stavares843 
Finally I put localhost as you requested.
There is an easy way to pass url param while starting cypress like this: $(node_modules location)/node_modules/.bin/cypress run --spec "cypress/integration/e2eTests/**/**/" --config baseUrl=http://localhost:3000/ --headed
we must just add baseUrl=xxx param
We can test in githubActions:
config:
          baseUrl=http://localhost:3000 or
env:
          CYPRESS_baseUrl: http://localhost:3000

https://github.com/cylc/cylc-ui/actions/runs/54043029/workflow